packages/scripts/src/testnet/OptimisticOracleRequestSender.ts

```diff
@@ -0,0 +1,80 @@
+
+const approvedIdentifier = "0x5945535f4f525f4e4f5f51554552590000000000000000000000000000000000";
+
+const goerliUsdc = "0x07865c6E87B9F70255377e024ace6630C1Eaa37F";
+
+const requestTime = Math.floor(new Date().valueOf() / 1000) - 1000;
+
+async function main() {
+  console.log("Running OptimisticOracleRequestSender script 🚀");
+
+  const chainId = (await ethers.provider.getNetwork()).chainId;
+
+  if (!chainId || chainId != 5) throw new Error("This script should be run on goerli");
+
+  //   OO v1.
+  const optimisticOracle = await getContractInstance<OptimisticOracleEthers>("OptimisticOracle", undefined, 5);
+  console.log("Sending requests to OptimisticOracle", optimisticOracle.address);
+  const ooTx = await optimisticOracle.requestPrice(approvedIdentifier, requestTime, ancillaryData, goerliUsdc, "0");
+  ooTx.wait();
+  console.log("Sent request to OptimisticOracle", ooTx.hash);
+
+  //   OO v2.
+  const optimisticOracleV2 = await getContractInstance<OptimisticOracleV2Ethers>("OptimisticOracleV2", undefined, 5);
+  console.log("Sending requests to OptimisticOracleV2", optimisticOracleV2.address);
+  const ooV2Tx = await optimisticOracleV2.requestPrice(approvedIdentifier, requestTime, ancillaryData, goerliUsdc, "0");
+  ooV2Tx.wait();
+  console.log("Sent request to OptimisticOracleV2", ooV2Tx.hash);
+
+  //   OO Skinny.
+  const skinnyOptimisticOracle = await getContractInstance<SkinnyOptimisticOracleEthers>(
+    "SkinnyOptimisticOracle",
+    undefined,
+    5
+  );
+  console.log("Sending requests to SkinnyOptimisticOracle", skinnyOptimisticOracle.address);
+  const ooSkinnyTx = await skinnyOptimisticOracle.requestPrice(
+    approvedIdentifier,
+    requestTime,
+    ancillaryData,
+    goerliUsdc,
+    "0",
+    "0",
```

Comment:
not sure if these are bond values, but would be really useful if we could propose with bonds that we could easily get for dev wallets